src/kubectl.rs

```diff
@@ -248,11 +249,27 @@ impl Kubectl {
                 // Wait for the child process to finish
                 let status = child.wait();
                 let status = match status {
-                    Ok(status) => status,
+                    Ok(status) => {
+                        match status.code().unwrap() {
+                            1 => {
+                                // Kill the process if the exit code is 1
+                                out_tx
+                                    .send(ChildEvent::Exit(
+                                        id,
+                                        status,
+                                        RestartPolicy::WillRestartIn(RetryDelay::NEVER)
```

Comment:
That line now reads that it will restart, but won't. Please change it to use `RestartPolicy::None` instead to indicate that there is no restart.

Reply:
My bad, misunderstood the usage here. Fixed with [e915a6c](https://github.com/sunsided/k8sfwd/pull/12/commits/e915a6c483e5f65da6fddfad1b346300e7c621e0)